src/test/java/org/gnode/nix/TestDataArray.java

```diff
@@ -0,0 +1,178 @@
+package org.gnode.nix;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class TestDataArray {
+
+    private File file;
+    private Block block;
+    private DataArray array1, array2;
+
+    private Date statup_time;
+
+    @Before
+    public void setUp() {
+        // precision of time_t is in seconds hence (millis / 1000) * 1000
+        statup_time = new Date((System.currentTimeMillis() / 1000) * 1000);
+
+        file = File.open("test_DataArray.h5", FileMode.Overwrite);
+
+        block = file.createBlock("block_one", "dataset");
+        array1 = block.createDataArray("array_one",
+                "testdata",
+                DataType.Double,
+                new NDSize(new int[]{0, 0, 0}));
+        array2 = block.createDataArray("random",
+                "double",
+                DataType.Double,
+                new NDSize(new int[]{20, 20}));
```

Comment:
Looking at the use of `createDataArray` I wonder whether it might be better to have int[] directly in the signature instead of `NDSize`. However, we can still do this later.